### ISSUE TITLE
TinaAdmin + Chrome

### ISSUE THREAD

**Issue by Enigmatical** [tinacms/tinacms#2421]
**Ready once https://github.com/tinacms/tinacms/pull/2261 is merged into `main`**

## Overview

@spbyrne and I had a call today to discuss what happens when `inject-tailwind` is merged and what the next steps will be.  For the most part, none of these appear to be terribly complicated and I've compiled a list of known tasks split between **Sidebar** and **TinaAdmin**.

### Changes to Sidebar (#2474 , PR https://github.com/tinacms/tinacms/pull/2434)

* [x] Move the default  **Document Creator** (`content-creator` Plugin) to the Navigation
  * It will only show when `tina-admin` flag is `false` (i.e. not shown when `TinaAdmin` is enabled)
  * If not possible, we could list out `content-creator` plugins in a new location - `TinaAdmin`'s Dashboard, for example.
* [x] Replace old `GlobalForm` menu with new Tina Navigation
  * For Phase 1, we will omit the User Information part.
  * [x] "Logout" action will live under the Navigation header's triple-dot menu.
* [x] Fullscreen Toggle
  * Only available for live content-editing (Preview)
* [x] General design changes to the Sidebar handles
  * Floats at the top rather than the bottom, pencil instead of arrow icon, etc.
* [x] Changes to the Sidebar header
  * Link to navigate to the Collection List page in `TinaAdmin`
  * "Edit `collection.name`"
  * Unsaved Changes indicator (!)
  * Removal of "Edit in TinaAdmin" links
* [ ] Reference Field "Edit in TinaAdmin" link revision
  * Does it route to the mapped Preview page if it exists?
  * Need to explore this.

### Changes to TinaAdmin (https://github.com/tinacms/tinacms/issues/2464, PR https://github.com/tinacms/tinacms/pull/2506)

* [x] Bring over the new Tina Navigation to replace old Navigation menu
* [x] Updates to the Collection List page
  * Updated header and placement of "Create New"
  * Wider list with more space for items to "breathe"
  * Each list item:
    * Direct link to Live Preview from the item, if Preview exists
    * "Edit in Admin" under triple-dot menu for each item
      * "Delete" would also live under this menu (not planned for Phase 1)
* [x] Updates to the Collection Create & Update pages
  * Updated header with "Save" and "Reset" buttons moved to footer
  * Unsaved Changes indicator

### Updates to Starters
* [x] Update `experimental-examples` and `examples` in `tinacms` to use `tina-admin` flag to `true` by default
* [x] Update `tina-docs-starter` to default `tina-admin` flag to `true` by default
* [x] Update `tina-barebones-starter` to default `tina-admin` flag to `true` by default
* [x] Update `tina-cloud-starter` to default `tina-admin` flag to `true` by default

### Updates to Documentation
* The changes to flow are documented pretty well here: https://whimsical.com/tina-chrome-mvp-WgrW8kcx5QoBNmi5JS6tQ1
* Essentially, all roads lead to `TinaAdmin`.  `RouteMapper` will become more important as it is the link between a list of `Document`s and their `Preview Editor` views.

**Comment by spbyrne** [comment]
I have most of the "Changes to Sidebar" task list done on the `chrome-components` branch, just started working on the last item.

<img width="1267" alt="Screen Shot 2022-01-14 at 11 00 25 AM" src="https://user-images.githubusercontent.com/5075484/149536698-c0dc4936-8b34-4759-9d53-782c4207d511.png">

**Comment by Enigmatical** [comment]
Took a video to show our progress so far (I hope @spbyrne doesn't mind).  This is a majority of his work in restyling with me handling the `<FormStatus />` and "Log Out" functionality.  Even after a couple of days, the difference between "old" Tina and "new" Tina is stunning:

https://www.loom.com/share/7f8c3918b2b3467189394d1d2dfbcca6
